src/sdk/bucketspace_fs.js

```diff
@@ -650,6 +623,73 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
     async put_object_lock_configuration(params, object_sdk) {
         // TODO
     }
+
+    /////////////////
+    ///// UTILS /////
+    /////////////////
+
+    // TODO: move the following 3 functions - has_bucket_action_permission(), validate_fs_bucket_access(), _has_access_to_nsfs_dir()
+    // so they can be re-used
+    async has_bucket_action_permission(bucket, account, action, bucket_path = "") {
```

Comment:
Why changing the name of the parameter from `method` to `action`?

Reply:
https://github.com/noobaa/noobaa-core/blob/master/src/server/common_services/auth_server.js#L673
it wasn't changed